### PR TITLE
 UI/UX adjustments

### LINKED ISSUE #4249: UI/UX Adjustments
- [x] Add "Nutanix" on the Provider Settings side of (4) Initial Nodes in Cluster Create
- [x] #4250
image

- [x] Force icons to stay in one line
image

- [x] Fix Event table line breaks
image

### PR DESCRIPTION
**Pull request by ahmadhamzh**
### What this PR does / why we need it
Add "Nutanix" on the Provider Settings side of (4) Initial Nodes in Cluster Create,
Adjust number input styling,
Force icons to stay in one line in machine deployments card,
Fix Event table line breaks width 
 
“closes #4249 ”

```release-note
NONE
```

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@ahmadhamzh If you want to close the issue automatically when it will be merged, write `closes #4249` or `fixes #4249` instead of just issue number.

**Comment by maciaszczykm** [comment]
Zrzut ekranu 2022-03-9 o 09 17 11

Zrzut ekranu 2022-03-9 o 09 17 48

@ahmadhamzh If you take a closer look the `-` button has wrong styling (border radius).

File: src/assets/css/global/_main.scss [hunk 1]
```diff
@@ -395,6 +395,10 @@ hr {
       }
     }
 
+    input {
+      text-align: right;
```
Comment (maciaszczykm) [inline src/assets/css/global/_main.scss:399]: Why do we need it?

Comment (ahmadhamzh) [reply src/assets/css/global/_main.scss:399]: so the number showed to the right like in the pic 

156377386-ab8940d3-bffe-408f-aa2c-3e5c59ecb087

File: src/app/shared/components/number-stepper/style.scss [hunk 2]
```diff
@@ -31,8 +31,7 @@ input[type='number']::-webkit-outer-spin-button {
     margin: 4px 0 0;
 
     &.left {
-      border-bottom-right-radius: 0;
-      border-top-right-radius: 0;
+      border-radius: 0;
```
Comment (maciaszczykm) [inline src/app/shared/components/number-stepper/style.scss:34]: Isn't that default?

File: src/assets/css/global/_main.scss [hunk 3]
```diff
@@ -395,13 +395,18 @@ hr {
...
+
     .mat-form-field-infix {
       width: 100%;
     }
 
     .mat-form-field-outline-start,
     .mat-form-field-outline-end {
-      border-radius: 0;
+      border-bottom-right-radius: 0;
+      border-top-right-radius: 0;
```
Comment (maciaszczykm) [inline src/assets/css/global/_main.scss:409]: Check if excluding `.mat-form-field-outline-start` will not solve it too. Then you could keep `border-radius: 0;` as the only rule.

File: src/app/shared/components/number-stepper/style.scss (hunk 2 above)
Comment (ahmadhamzh) [reply src/app/shared/components/number-stepper/style.scss:34]: no it's not if i remove it all the corner will have a radius 
Screenshot from 2022-03-10 12-21-16

**Pull request merged**